components/navbar/CartButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { faShoppingCart } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { CartPopover, useCart } from '@reservoir0x/reservoir-kit-ui'
+import { Flex, Button, Text } from 'components/primitives'
+
+const CartButton = () => {
+  const { data: cartItems } = useCart((cart) => cart.items)
+
+  return (
+    <CartPopover
+      trigger={
+        <Button
+          css={{
+            justifyContent: 'center',
+            width: '44px',
```

Comment:
I think we also need to set the height to 44px here. The button is a little taller than the other navbar buttons